# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/notification/NotificationParent.h
# Commit: a497ddb030f6
# Full Hash: a497ddb030f6d0852bca56f875b48bc2e1ebc134
# Author: Kagami Sascha Rosylight <saschanaz@outlook.com>
# Date: 2024-11-08 04:19:40
# Regressor Bug: 1928702
# File Overlap Count: 1
# Description:
#   Bug 1928702 - Part 5: Implement RecvShow r=asuth
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D227690
# ==============================================================================

diff -r 5a94b945a3ad -r a497ddb030f6 dom/notification/NotificationParent.h
--- a/dom/notification/NotificationParent.h	Thu Nov 07 17:32:05 2024 +0000
+++ b/dom/notification/NotificationParent.h	Thu Nov 07 17:32:05 2024 +0000
@@ -45,6 +45,8 @@
  private:
   ~NotificationParent() = default;
 
+  nsresult Show();
+
   void GetAlertName(nsAString& aRetval) {
     if (mAlertName.IsEmpty()) {
       MaybeInitAlertName();